### DISCUSSION TITLE
How to get clipboard data without ctrl v?

### DISCUSSION THREAD

**Discussion by imxood** [emilk/egui#3281] [Q&A]
How to get clipboard data without ctrl v?

**Comment by YgorSouza** [comment]
This is currently not implemented as far as I can tell. I tried to implement it as a test and it seems to be feasible, although the clipboard logs an error and returns `None` while the window is being dragged. I don't know if this can be avoided and if there are other undesirable side effects.

```
ERROR egui_winit::clipboard] smithay paste error: no focus on a seat.
```

**Reply by imxood** [reply]
Finally, I directly used arboard crate...

```rust
use arboard::Clipboard;

let clipboard = Clipboard::new().unwrap();

clipboard.set_text(text);

clipboard.get_text();
```
